src/signals/incident/components/form/ContainerSelect/WfsLayer/WfsLayer.tsx

```diff
@@ -1,51 +1,67 @@
-import type { FunctionComponent } from 'react';
+import React, { useContext, useEffect, useState } from 'react';
 import { useMapInstance } from '@amsterdam/react-maps';
 import type { FeatureCollection } from 'geojson';
-import React, { useContext, useEffect, useState } from 'react';
 import { MAX_ZOOM_LEVEL, MIN_ZOOM_LEVEL } from '@amsterdam/arm-core/lib/constants';
-import type { DataLayerProps, WfsLayerProps } from '../types';
+import type { DataLayerProps } from '../types';
+import L from 'leaflet';
+import type { Map as MapType } from 'leaflet';
 
 import { fetchWithAbort } from '@amsterdam/arm-core';
 import type { ZoomLevel } from '@amsterdam/arm-core/lib/types';
 import ContainerSelectContext from '../ContainerSelectContext';
 import { NO_DATA, WfsDataProvider } from './WfsDataContext';
 
-export const isLayerVisible = (zoom: number, zoomLevel: ZoomLevel) => {
+const SRS_NAME = 'urn:ogc:def:crs:EPSG::4326';
+const DEFAULT_ZOOM_LEVEL: ZoomLevel = {
+  max: 7,
+};
+
+export const isLayerVisible = (zoom: number, zoomLevel: ZoomLevel): boolean => {
   const { min, max } = zoomLevel;
   return zoom <= (min ?? MIN_ZOOM_LEVEL) && zoom >= (max ?? MAX_ZOOM_LEVEL);
 };
 
-export interface Props extends WfsLayerProps {
+export interface WfsLayerProps {
   children: React.ReactElement<DataLayerProps>;
+  zoomLevel?: ZoomLevel;
 }
 
-const WfsLayer: FunctionComponent<Props> = ({ url, options, zoomLevel, children }) => {
+const WfsLayer: React.FC<WfsLayerProps> = ({ children, zoomLevel = DEFAULT_ZOOM_LEVEL }) => {
```

Comment:
I would prefer to keep using FunctionComponent here as this is the current trend.